src/index.js

```diff
@@ -10,6 +10,7 @@ import stack from './lib/stack'
 import BarView from './components/Bar'
 
 const APP_SELECTOR = '[role=application]'
+const BANNER_SELECTOR = '[role=banner]'
```

Comment:
Yes, you're definitively right. I hit an issue with a missing element, so I added the test on every element, before understanding that the banner was injected by our code.